es-core/src/Window.cpp

```diff
@@ -404,34 +409,83 @@ bool Window::isProcessing()
 }
 
 void Window::startScreenSaver()
- {
- 	if (mScreenSaver && !mRenderScreenSaver)
- 	{
- 		// Tell the GUI components the screensaver is starting
- 		for(auto i = mGuiStack.begin(); i != mGuiStack.end(); i++)
- 			(*i)->onScreenSaverActivate();
-
- 		mScreenSaver->startScreenSaver();
- 		mRenderScreenSaver = true;
- 	}
- }
-
- void Window::cancelScreenSaver()
- {
- 	if (mScreenSaver && mRenderScreenSaver)
- 	{
- 		mScreenSaver->stopScreenSaver();
- 		mRenderScreenSaver = false;
-
- 		// Tell the GUI components the screensaver has stopped
- 		for(auto i = mGuiStack.begin(); i != mGuiStack.end(); i++)
- 			(*i)->onScreenSaverDeactivate();
- 	}
- }
-
- void Window::renderScreenSaver()
- {
- 	if (mScreenSaver)
- 		mScreenSaver->renderScreenSaver();
- }
+{
+	if (mScreenSaver && !mRenderScreenSaver)
+	{
+		// Tell the GUI components the screensaver is starting
+		for(auto i = mGuiStack.begin(); i != mGuiStack.end(); i++)
+			(*i)->onScreenSaverActivate();
+
+		mScreenSaver->startScreenSaver();
+		mRenderScreenSaver = true;
+	}
+}
+
+void Window::cancelScreenSaver()
+{
+	if (mScreenSaver && mRenderScreenSaver)
+	{
+		mScreenSaver->stopScreenSaver();
+		mRenderScreenSaver = false;
+
+		// Tell the GUI components the screensaver has stopped
+		for(auto i = mGuiStack.begin(); i != mGuiStack.end(); i++)
+			(*i)->onScreenSaverDeactivate();
+	}
+}
+
+void Window::renderScreenSaver()
+{
+	if (mScreenSaver)
+		mScreenSaver->renderScreenSaver();
+}
```

Comment:
Note it would be preferable to have these whitespace fixes above done as a separate commit in the PR (cleaner, more clear to see what is changed etc). These indent errors look like they were introduced with https://github.com/RetroPie/EmulationStation/commit/59d7516a1618604644067101aed87a4cc13f3fe0 - I must have missed them. tagging @pjft for reference.

Reply:
Thanks for tagging. I'll be mindful of those in the future - I believe this was before I had made sure to set up my IDE to handle the white spaces and tabs accordingly, but will double check, especially as I move to a new set up.